frontend/webservice/node.cgi

```diff
@@ -196,4 +209,16 @@ sub get_nodes {
     return { results => $result };
 }
 
+sub delete_node {
+    my $method = shift;
+    my $params = shift;
+
+    my $err = OESS::DB::Node::decom(db => $db, node_id => $params->{node_id}{value});
+    if(defined $err){
+       $method->set_error($err);
+        return;
+    }
+    return { results => ''};
```

Comment:
For successes like these return `{ results => [{ success => 1 }] }`. I personally don't like the format but that's kinda what we've settled on.